atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -175,15 +178,15 @@ public int execute(final AtlasDbServices services) {
                         ? sweepRunner.dryRun(tableToSweep, batchConfig, startRow.get())
                         : sweepRunner.run(tableToSweep, batchConfig, startRow.get());
                 printer.info(
-                        "Swept from {} to {} in table {} in {} ms, examined {} cell values,"
-                                + " {}deleted {} stale versions of those cells.",
-                        encodeStartRow(startRow),
-                        encodeEndRow(results.getNextStartRow()),
-                        tableToSweep,
-                        watch.elapsed(TimeUnit.MILLISECONDS),
-                        results.getCellTsPairsExamined(),
-                        dryRun ? "would have " : "",
-                        results.getStaleValuesDeleted());
+                        "{} Swept from {} to {} in table {} in {} ms, examined {} cell values,"
+                                + " deleted {} stale versions of those cells.",
+                        SafeArg.of("isDryRun", dryRun ? "[DRY RUN]" : ""),
+                        UnsafeArg.of("startRow", encodeStartRow(startRow)),
+                        UnsafeArg.of("exclusiveEndRow", encodeEndRow(results.getNextStartRow())),
+                        LoggingArgs.tableRef(tableToSweep),
+                        SafeArg.of("time taken", watch.elapsed(TimeUnit.MILLISECONDS)),
```

Comment:
nit: should we give the units, given that we've started shifting towards preferring microseconds internally?